### ISSUE TITLE
Userspace RCU doesn't pass the tests on SLES 12 x86_64 on AT 11.0

### ISSUE THREAD

**Issue by tuliom** [advancetoolchain/advance-toolchain#30]
Needs to investigate and fix this error:

`
urcu: x86_64:  Start logging...
urcu: x86_64:  Testing 64 bit static linking...
urcu: x86_64:  Testing 64 bit dynamic linking...
urcu: x86_64:  Unrecognized error: /opt/at-next-11.0-0-alpha/bin/powerpc64le-linux-gnu-ld: warning: liburcu-common.so.4, needed by /opt/at-next-11.0-0-alpha/ppc64le/usr/lib/../lib64/liburcu.so, not found (try using -rpath or -rpath-link)
`

**Comment by racardoso** [comment]
@tuliom can you give more details about how to reproduce this error? Like at version, make command.

**Comment by tuliom** [comment]
@racardoso Sure! It's happening on AT next.  You can reproduce it if you use remote-build.exp on a SLES 12 x86_64 cross compiling to ppc64le.
Are you having trouble to reproduce it?

**Comment by racardoso** [comment]
@tuliom I was trying using a stable version.

**Comment by racardoso** [comment]
Still can't reproduce the problem:

Run `./remote-build.exp --buildarch=ppc64le next SLES_X86_64`

Fails test `check_package`:

```
Completed RPM packages for at12.0-0-alpha
Running FVTR for at12.0-0-alpha...
amino                   x86_64:         ignored
boost                   x86_64:         passed
ck_binaries             x86_64:         passed
ck_gold                 x86_64:         passed
ck_ldconfig             x86_64:         ignored
ck_ldds                 x86_64:         ignored
ck_libbits              x86_64:         passed
ck_lock_elision         x86_64:         ignored
ck_provides             x86_64:         ignored
ck_requires             x86_64:         passed
ck_tuned                x86_64:         ignored
dhrystone               x86_64:         ignored
gcc-builtins            x86_64:         passed
gccgo                   x86_64:         passed
gdb                     x86_64:         passed
gmp                     x86_64:         ignored
golang                  x86_64:         ignored
gotools                 x86_64:         ignored
libauxv                 x86_64:         passed
libdfp                  x86_64:         passed
libexpat                x86_64:         ignored
libhugetlbfs            x86_64:         passed
libvecpf                x86_64:         passed
mpfr                    x86_64:         ignored
openssl                 x86_64:         passed
oprofile                x86_64:         ignored
package-check           x86_64:         failed
paflib                  x86_64:         passed
ppc476-workaround       x86_64:         ignored
pthreads                x86_64:         passed
python                  x86_64:         ignored
speccpu2006             x86_64:         ignored
sphde                   x86_64:         passed
sysroot                 x86_64:         passed
tbb                     x86_64:         passed
tcmalloc                x86_64:         passed
timezone                x86_64:         ignored
urcu                    x86_64:         passed
valgrind                x86_64:         ignored
xlc                     x86_64:         ignored
xlcdfp                  x86_64:         ignored
xlf                     x86_64:         ignored
zlib                    x86_64:         passed

**************** TEST SUMMARY *****************
Passed tests: 21
Skipped tests: 21
Failed tests: 1
***********************************************

```

Notice that ucru test passed.

**Comment by racardoso** [comment]
Confirmed AT 11.0:
`urcu: x86_64:  Unrecognized error: /home/rcardoso/opt/at11.0-0-alpha1/bin/powerpc64le-linux-gnu-ld: warning: liburcu-common.so.4, needed by /home/rcardoso/opt/at11.0-0-alpha1/ppc64le/usr/lib/../lib64/liburcu.so, not found (try using -rpath or -rpath-link`

**Comment by racardoso** [comment]
Closed via  d2f3cd0
